docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+# On container startup log in with username and password if given and start the server
+CMD if [ -n ${USERNAME} -a -n ${PASSWORD} ]; then \
+        supertuxkart --init-user --login=${USERNAME} --password=${PASSWORD}; \
+    fi && \
+    supertuxkart --server-config=server_config.xml
```

Comment:
The `CMD` copied from the Dockerfile has to be removed. You also don't need the `\` and `&&` since this is a not an inline script anymore.